assets/src/edit-story/app/media/mediaProvider.js

```diff
@@ -143,6 +157,9 @@ function MediaProvider({ children }) {
   );
 
   const generatePoster = useCallback(() => {
+    // eslint-disable-next-line no-shadow
```

Comment:
This is just an effect and IMHO should be inlined as an effect. The effect itself is already dependent on the `media`, so no reason to put it into a `ref`. The `generatePoster` shouldn't be a public API at all, hence its dependencies do not matter.

Reply:
This change has been reverted.